airflow/api_fastapi/core_api/routes/public/dag_tags.py

```diff
@@ -34,13 +35,14 @@
 )
 from airflow.api_fastapi.common.router import AirflowRouter
 from airflow.api_fastapi.core_api.datamodels.dag_tags import DAGTagCollectionResponse
+from airflow.api_fastapi.core_api.security import ReadableTagsFilterDep, requires_access_dag
 from airflow.models.dag import DagTag
 
 dag_tags_router = AirflowRouter(tags=["DAG"], prefix="/dagTags")
 
 
 @dag_tags_router.get(
-    "",
+    "", dependencies=[Depends(requires_access_dag(method="GET", access_entity=DagAccessEntity.CODE))]
```

Comment:
Why do we set `access_entity` to `DagAccessEntity.CODE` here?

Reply:
Should we just use `Depends(requires_access_dag(method="GET")` or do we need to create a new entity?

Reply:
I think, we need to add new entity TAGS

Reply:
I would say that a new entity might be overkill.

Dags information contains tag anyway. So let's go for Dag.

If later we can edit / create tag without going through the dag entity maybe a new entity will make sense then

Reply:
Yep, I also feel that way. Just updated it